example/layer-examples.js

```diff
@@ -166,6 +167,34 @@ export function ArcLayer64Example(props) {
   });
 }
 
+export function ChoroplethLayer64ContourExample(props) {
+  const {mapViewState, choropleths} = props;
+  return new ChoroplethLayer64({
+    id: props.id || 'choroplethContourLayer64',
+    width: window.innerWidth,
+    height: window.innerHeight,
+    ...mapViewState,
+    data: choropleths,
+    opacity: 0.8,
+    drawContour: true
+  });
+}
+
+export function ChoroplethLayer64SolidExample(props) {
+  const {mapViewState, choropleths} = props;
+  return new ChoroplethLayer({
+    id: props.id || 'choroplethLayer',
+    width: window.innerWidth,
+    height: window.innerHeight,
+    ...mapViewState,
+    data: choropleths,
+    opacity: 0.01,
+    isPickable: true,
+    onHover: props.onChoroplethHovered,
+    lick: props.onChoroplethClicked
```

Comment:
Yeah... some typos from the original choropleth layer got copied to the choropleth64... Thanks for your sharp eyes!